pkg/storageops/storageops.go

```diff
@@ -0,0 +1,74 @@
+package storageops
+
+const (
+	// SetIdentifierNone is an default identifier to group all disks managed by
+	// storage ops
```

Comment:
nit: is a default identifier to group all disks from a particular set

Reply:
done